splunk_eventgen/lib/plugins/output/syslogout.py

```diff
@@ -8,16 +8,24 @@
 # Dict of flags to gate adding the syslogHandler only once to the given singleton logger
 loggerInitialized = {}
 
+class HostFilter(logging.Filter):
+    def __init__(self, host):
+        self.host = host
+
+    def filter(self, record):
```

Comment:
This filter always returns `True`.

Reply:
Correct, this is as intended - the filter should never return False, because all events should be sent. The purpose of the filter is not to remove messages from the output stream, it is only to add the host field so it's available in the formatter further down.

Reply:
please add this description as comment in HostFilter::filter method.
It would be much easier for other people to understand the purpose of this filter.